Translations: Revert recent changes to pt_BR

rffontenelle opened the pull request:
The contributor applied changes only to `pcsx2_Main.po`. By not changing `pcsx2_Iconized.po` as well caused translation inconsistency between `pcsx2_Main.po` and `pcsx2_Iconized.po`, which has not being solved for 3 months now.

For the record, the reverted commits contain predominantly (good) style changes, not about gross error or typo fixes of previous translations.

pgert commented:
@GuilhermeDiasJ:
Any comments?

GuilhermeDiasJ commented:
> @GuilhermeDiasJ:
> Any comments?

Bad translations, bad grammar. This guy shouldn't mess with translations. I don't recommend accepting his translations, I fixed every typo and now he added them back.

GuilhermeDiasJ commented:
>"Audio", "Resumir", "Save", "Agressivo plus", "Sub-filtro", Title-case translations, etc.
Yeah, congratulations on turning the Portuguese translation into a mess. No changes made by me were invalid, as I fixed all possible grammar errors and changed some terms that are most commonly used in Portuguese translations in general. You can't undo all my work just because you don't agree with the changes.

rffontenelle commented:
Wow, hold your horses. "This guy" having been translating PCSX2 since 2011, both UI and User Guide, and is also coordinator of translation in some big projects and contributor in several other open and free source projects. My English and Portuguese are fine, thanks. So, think again what you are saying.

There is no need for personal attack against "this guy". I didn't criticized your changes -- some were good fixes, a few are wrong, and most are your personal choice/style. However, most important of all, your commits are incomplete because you changed only part of the UI (`pcsx2_Main.po`), causing inconsistency and it should only land the master branch when you apply the changes in both file.

I notice you still have your fork. Keep working in your fork, apply the changes to `pcsx2_Iconized.po` as well and then reapply with both files. Done.

On the mentioned "errors":
- **&Audio** doesn't have accent in "A" because & is a mnemonic character and Á breaks it (No sure, though; long time I translated it)
- **Resumir**: I preferred your translation
- **Save** on savestate can be perfectly translated to "salvar". Personally I think "guardar" is a little bit weird, but ok. But, pcsx2_Iconized.po still has "Salvar" so we have inconsistency
- **Agressivo plus**: it was a conscientious choice, but I like your "Mais agressivo"
- **sub-filtro** is the correct translation to subfilter. [Post-OA1990](https://en.wikipedia.org/wiki/Portuguese_Language_Orthographic_Agreement_of_1990), "sub" has hyphen only if next latter is "h", "r" and "b".
- **Title-case translations**: English uses title-case text for labels and titles, while Brazilian Portuguese uses sentence case since 2009 due to the [Portuguese Language Orthographic Agreement of 1990](https://en.wikipedia.org/wiki/Portuguese_Language_Orthographic_Agreement_of_1990) (on the [base XIX](http://www.portaldalinguaportuguesa.org/acordo.php?action=acordo&version=1990))

GuilhermeDiasJ commented:
My apologies for the inconvenience, I admit I overreacted in my comments and behaviour, I got mad because you literally undid all my edits, switching to an old version that contains many grammatical mistakes, but let's try to solve that:
I rarely use github, so I didn't know if it was really necessary to edit the other file, as my only interest was to improve the translation of the UI interface.
I'll try, but I have no experience in that. The last time I updated this branch I had to edit a lot of things in order to upload it, and with the help of other users here. 
- **Audio**: Actually it works, newer versions that uses the current translation doesn't have any issues.
- **Save**: "Salvar" is fine too, but I was actually referring to the noun in some strings, which didn't translate it when it should be "salvamento" or "gravação".
- **Sub-filtro**: Then it should be "Subfiltro", right? The hyphen is added only if next letter is "h", "r" or "b", and not letter "f".
- **Title-case translations**: Exactly, but your changes actually brought back some of them that I had fixed some time ago.

Ok, I didn't want to remove your credits because you already worked it before and I wasn't translating from scratch.
But at least you could have edited a few things, right? The revision you reverted has many mistakes.

rffontenelle commented:
> My apologies for the inconvenience, I admit I overreacted in my comments and behavior, I got mad because you literally undid all my edits, switching to an old version that contains many grammatical mistakes, but let's try to solve that:

No problem, let's move on.

> I rarely use github, so I didn't know if it was really necessary to edit the other file, as my only interest was to improve the translation of the UI interface.

The need for editting both `pcsx2_Main.po` and `pcsx2_Iconized.po` is not related to GitHub itself. Once the commit hits the master branch, it will be available for anyone who builds PCSX2 from latest revision (e.g. [pcsx2-git](https://aur.archlinux.org/packages/pcsx2-git/) in Arch Linux AUR, which always get latest revision in buildtime). This PCSX2 built will receive the translation from the files currently available in the master branch.

So, if e.g. "savestate" translation is changed to "_estado de jogo guardado_" in `pcsx2_Main.po` but not in `pcsx2_Iconized.po`, the UI will show labels with "_estado de jogo guardado_" but some tooltip will still have "_estado de jogo salvo_" -- hence the inconsistency I mentioned.

> I'll try, but I have no experience in that. The last time I updated this branch I had to edit a lot of things in order to upload it, and with the help of other users here.

If you want, we can work together, like via Telegram or whatever, and I can give you further assistance. PM me (josephg) in PCSX2 forums.

Let me step back and bring up some Git-related info:

- With [git revert](https://git-scm.com/docs/git-revert) command, one can basically undo a specific commit. It is not a manual edition of the files, instead it is like a return a previous snapshot of that file. So, I had the option of simply temporarily removing your commits from master branch (for later you re-add), or I could go through the whole file manually to undo. Since both options looks like "aggressively undoing the work you put through" and the second one would cost me more time, I chose the first option ;)

- When you edit `pcsx2_Main.po` via GitHub web interface, GitHub creates for you a fork of PCSX2 (https://github.com/GuilhermeDiasJ/pcsx2) and each edit you save creates a branch "patch-NUM" (e.g. patch-1, patch-2, patch-3).

- Notice that applying this git-revert will simply remove your changes from PCSX2 official repository, but you still have the changes in your fork. So your effort will not be completely lost, unless you simply remove your fork, which I don't recommend.

- Since you still have your changes in your fork, now you should work on `pcsx2_Iconized` to reflect the same changes you made to `pcsx2_Main.po` (e.g. "estado de jogo guardado" in both) in order to keep consistency.

- Once you are done changing the translations, submit both files via one Pull Request, preferably in a single commit with both files.

- Consider to test your translation in a local build of PCSX2, so you can double-check if your changes looks good in the result GUI.

> 
> **Audio**: Actually it works, newer versions that uses the current translation doesn't have any issues.

Just checking: Pressing Alt+A selects "&Áudio", even with the "Á" as mnemonic? If yes, then I'm fine with it.
 
> **Save**: "Salvar" is fine too, but I was actually referring to the noun in some strings, which didn't translate it when it should be "salvamento" or "gravação".
> 

Now I understood. I double-checked and I prefer your suggestion.

> **Sub-filtro**: Then it should be "Subfiltro", right? The hyphen is added only if next letter is "h", "r" or "b", and not letter "f".

Oh I see, I did the opposite I just explained. Ok, "subfiltro" it is.
 
> **Title-case translations**: Exactly, but your changes actually brought back some of them that I had fixed some time ago.

I see. Maybe I missed some of them last time I applied (at least tried) sentence case to all labels.
 
> Ok, I didn't want to remove your credits because you already worked it before and I wasn't translating from scratch.

By doing what I mentioned, you wouldn't be removing my credits. The header entry `Last-Translator` should always be the last contributor that translated that file, and `PO-Revision-Date` should be the date of the last translation. So please always update them when you edit the file.

Also consider adding your credit by inserting/updating a comment line with your name, e-mail and year of contribution, right below mine, and without removing mine, in one of the first lines of the gettext PO file:
```
# Guilherme Dias <guilhermedias000@gmail.com>, 2019
```

> But at least you could have edited a few things, right? The revision you reverted has many mistakes.

As mentioned above, "git revert" is all or nothing.

**Finally, @GuilhermeDiasJ, can we go ahead with merging this Pull Request, and you continue to fix the translation (both files) and then submit it again afterwards?**  As I mentioned, I'm fine helping you if you need Git- or translation-related help in another place; just ping me in PCSX2 forums

lightningterror commented:
What is the status on this?

rffontenelle commented:
> What is the status on this?

No update on my side.

pgert commented:
@rffontenelle:
Do you intend to close this PR and make a new one with alterations based on the existing translation?

rffontenelle commented:
@pgert Yep, I'll probably have time for that in a few days. Closing this PR.
